cli.js

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env node
 'use strict';
 
+// Prefer the local installation of AVA.
+var resolveFrom = require('resolve-from');
+var localCLI;
+try {
+	localCLI = resolveFrom('.', 'ava/cli');
+} catch (e) {}
+
+if (localCLI && localCLI !== __filename) {
+	console.warn('Using local install of AVA.');
```

Comment:
Maybe this is something we could have in a future `--verbose` mode or just add it as a `debug()`?